docs/content/en/docs/configuration/keys.md

```diff
@@ -1206,23 +1208,27 @@ See also:
 
 ## Configuration snippet
 
-| Configuration key    | Scope     | Default  | Since |
-|----------------------|-----------|----------|-------|
-| `config-backend`     | `Backend` |          |       |
-| `config-defaults`    | `Global`  |          | v0.8  |
-| `config-frontend`    | `Global`  |          |       |
-| `config-global`      | `Global`  |          |       |
-| `config-proxy`       | `Global`  |          | v0.13 |
-| `config-sections`    | `Global`  |          | v0.13 |
-| `config-tcp`         | `Global`  |          | v0.13 |
-| `config-tcp-service` | `TCP`     |          | v0.13 |
+| Configuration key       | Scope     | Default  | Since |
+|-------------------------|-----------|----------|-------|
+| `config-backend`        | `Backend` |          |       |
+| `config-defaults`       | `Global`  |          | v0.8  |
+| `config-frontend`       | `Global`  |          |       |
+| `config-frontend-early` | `Global`  |          | v0.15 |
+| `config-frontend-late`  | `Global`  |          | v0.15 |
+| `config-global`         | `Global`  |          |       |
+| `config-proxy`          | `Global`  |          | v0.13 |
+| `config-sections`       | `Global`  |          | v0.13 |
+| `config-tcp`            | `Global`  |          | v0.13 |
+| `config-tcp-service`    | `TCP`     |          | v0.13 |
 
 Add HAProxy configuration snippet to the configuration file. Use multiline content
 to add more than one line of configuration.
 
 * `config-backend`: Adds a configuration snippet to a HAProxy backend section.
 * `config-defaults`: Adds a configuration snippet to the end of the HAProxy defaults section.
 * `config-frontend`: Adds a configuration snippet to the HTTP and HTTPS frontend sections.
```

Comment:
Complete the sentence here saying that this is an alias to `config-frontend-late`.